provider/blockdev_snapshot_base.py

```diff
@@ -53,6 +53,10 @@ def prepare_snapshot_file(self):
             if self.params.get("mode") == "existing":
                 self.snapshot_image.create()
 
+    def data_disk_attached(self):
```

Comment:
We can do accurate judgement like:

self.params.get("base_tag") != "image1"

Reply:
have updated, please help to review again, thanks